pkgs/development/python-modules/opensfm/default.nix

```diff
@@ -121,6 +117,7 @@ buildPythonPackage rec {
   meta = {
     maintainers = [ lib.maintainers.SomeoneSerge ];
     license = lib.licenses.bsd2;
+    changelog = "${src.meta.homepage}/blob/${src.rev}/CHANGELOG.md";
```

Comment:
```suggestion
    changelog = "https://github.com/mapillary/OpenSfM/blob/${src.rev}/CHANGELOG.md";
```

Reply:
How is it any better if it's already using `src.rev`?

Reply:
Although... you're right, better not touch such internals